### DISCUSSION TITLE
Naive questions about theme JSON files

### DISCUSSION THREAD

**Discussion by vladstudio** [mikekazakov/nimble-commander#120] [Q&A] [answered]
1. what is `filePanelsGeneralOverlayColor` ? I tried setting it to bright pink, to identify where it's used, but have not find it used anywhere.

2. is toolbar styled with colors from JSON? Again, I tried changing all the colors but they do not seem to affect the toolbar.

<img width="1041" alt="image" src="https://github.com/mikekazakov/nimble-commander/assets/914320/c580589b-df41-4d05-a7e1-bd66dbe63b35">

3. is it necessary for Operations item in the toolbar to have different background and left/right borders? I believe the app would look better if Operations were "transparent", blended into the toolbar.

**Comment by mikekazakov** [comment, marked as answer]
1. Take a look: 
<img width="752" alt="Screenshot 2024-01-28 at 12 18 12" src="https://github.com/mikekazakov/nimble-commander/assets/7439287/75af6df7-1c6f-4062-9d8a-c04dd38a8840">

Admittedly it seems that this color is not immediately picked up.

2. Right now it's not customisable and uses the system-provided "windowBackgroundColor". Can be added into themes instead of course.

3. I don't think it's necessary, these colours can be customised as well. Could sketch how would it look with a transparent background and with no borders? BTW bear in mind there are also buttons there sometimes: up-down on the left and pause-cancel on the right.
